src/ExperimentTable.js

```diff
@@ -0,0 +1,193 @@
+        profile[tableHeader[this.state.searchedColumn].dataParam].toString().toLowerCase().includes(searchQuery.toLowerCase())
+      )
+  }
+
+  setValue(property){
+    return e => {
+      this.setState({
+        [property]: e.target.value,
+        currentPage: 1
+      })
+    }
+  }
+
+  handleCheckbox(accession){
+    const checkedArray = this.state.checkedArray.slice()
+    checkedArray.includes(accession) ?
+      _.remove(checkedArray, function(n) {return n === accession})
+      : checkedArray.push(accession)
+    this.setState({checkedArray: checkedArray})
+  }
+
+  render() {
+    const {selectedSearch, selectedKingdom, checkedArray, selectedNumber, currentPage, searchedColumn, searchQuery, orderedColumn, ordering} = this.state
+    const {host, aaData, tableHeader, enableDownload, enableIndex} = this.props
+
+    const dataArray = selectedSearch ? this.sort(aaData).filter(data => data && Object.values(data).some(value => value.toString().toLowerCase().includes(selectedSearch.toLowerCase()))) :
+      this.filter(this.sort(aaData), tableHeader).filter(data => selectedKingdom ? data.kingdom === selectedKingdom : true)
+    const currentPageData = selectedNumber ? dataArray.slice(selectedNumber*(currentPage-1), selectedNumber*currentPage) : dataArray
+
+    return (
+      <div>
+        <div className={`row expanded`}>
+          <div className={`large-2 medium-4 small-8 columns`}>
+            <label> Kingdom:
+              <select defaultValue={``} className={`kingdom`}
+                onChange={event => this.setState({selectedKingdom: event.target.value})}>
+                <option value={``} >All</option>
+                <option value={`plants`}>Plants</option>
+                <option value={`animals`}>Animals</option>
+                <option value={`fungi`}>Fungi</option>
+              </select>
+            </label>
+          </div>
+          <div className={`large-2 medium-4 small-8 columns`}>
+            <label>Entries per page:
+              <select defaultValue={10} onChange={this.setValue(`selectedNumber`)}>
+                <option value={10}>10</option>
+                <option value={25}>25</option>
+                <option value={50}>50</option>
+                <option value={100}>100</option>
+                <option value={aaData.length}>All</option>
+              </select>
+            </label>
+          </div>
+          <div className={`large-2 medium-4 small-8 columns`}>
+            <label>Search all columns:
+              <input type={`search`} placeholder={`Type here ...`}
+                onChange={this.setValue(`selectedSearch`)}/></label>
+          </div>
+        </div>
+
+        <Table border style={{display:`grid`}}>
+          <Table.Head>
+            {enableIndex && <Table.TextHeaderCell key={`index`} flexBasis={100} flexShrink={100} flexGrow={100}>Index</Table.TextHeaderCell>}
```

Comment:
What are these `flexBasis`, `flexShrink` and `flexGrow` props? Is this an Evergreen thing?

Reply:
Yes. `flexShrink` and `flexGrow` help to make the table responsive. `flexBasis` is the default width.

Reply:
I just found out that `flex` css has `flex-grow`, `flex-shrink`, and `flex-basis` setups to allow a flex item will grow or shrink to fit the space available in its flex container: https://developer.mozilla.org/en-US/docs/Web/CSS/flex, so Evergreen takes the usages of these properties.